### PR TITLE
(SERVER-893) Apply pl-clojure-style recommendations

### PR DESCRIPTION
**Pull request by justinstoller**
This commits the updates applied by running `lein cljfmt fix` using the
cljfmt alias provided in 5d896f2.

### CODE REVIEW

**Comment by justinstoller** [comment]
See #226 for more information.

**Comment by fpringvaldsen** [comment]
:+1:

**Comment by cprice404** [comment]
@justinstoller was this easy to do?  I ask because we have a bunch of PRs open against TK right now, and this one may cause merge conflicts with those.  If generating this PR is as simple as running the cli tool and then doing a commit, maybe we can close this and do it again as soon as the other PRs land?

File: src/puppetlabs/trapperkeeper/internal.clj [hunk 1]
```diff
@@ -346,13 +346,13 @@
   [shutdown-reason-promise :- IDeref
    app-context :- (schema/atom a/TrapperkeeperAppContext)]
   (s/service ShutdownService
```
Comment (pcarlisle) [inline src/puppetlabs/trapperkeeper/internal.clj]: you might want to add a custom rule for `service`

File: src/puppetlabs/trapperkeeper/services.clj [hunk 2]
```diff
@@ -66,36 +66,36 @@
           ;; the main service fnk for the app graph.  we add metadata to the fnk
           ;; arguments list to specify an explicit output schema for the fnk
           (fnk service-fnk# :- ~output-schema
```
Comment (pcarlisle) [inline src/puppetlabs/trapperkeeper/services.clj]: fnk maybe wants a rule too

**Comment by justinstoller** [comment]
@cprice404 yep, this was super easy to do. I can also look at adding the rules that patrick mentions (if there's consensus on adding them). And apply the formatting after the other PRs have landed.

**Comment by cprice404** [comment]
@justinstoller awesome, thanks.

**Comment by justinstoller** [comment]
Closing as outdated. See https://github.com/puppetlabs/trapperkeeper/pull/232 for an updated PR (and one that uses @pcarlisle recommendations). See https://github.com/puppetlabs/pl-clojure-style/pull/3 for the implementation of those recommendations.

**Pull request closed without merging**